projects/ion/src/lib/icon/icon.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { render } from '@testing-library/angular';
+import { IonIconComponent, IonIconProps } from './icon.component';
+
+const sut = async (customProps?: IonIconProps): Promise<HTMLElement> => {
+  let type = 'trash';
+
+  await render(IonIconComponent, {
+    componentProperties: customProps || {
+      type: 'trash',
+    },
+  });
+  customProps && (type = customProps.type);
+  return document.getElementById('ion-icon-' + type);
+};
```

Comment:
```suggestion
const sut = async (customProps: IonIconProps = { type: 'trash' }): Promise<HTMLElement> => {
  await render(IonIconComponent, {
    componentProperties: customProps,
    },
  });
  return document.getElementById('ion-icon-' + customProps.type);
};
```
I'm not sure if this alternative works properly...